jenkins/internal-ci/generic/nightly/nightly-deploy.groovy

```diff
@@ -24,6 +24,7 @@ pipeline {
         string(name: 'CORTX_ALL_IMAGE', defaultValue: 'cortx-docker.colo.seagate.com/seagate/cortx-all:2.0.0-latest', description: 'CORTX-ALL image', trim: true)
```

Comment:
Please remove CORTX-ALL image from the CORTX Deployment Jenkins pipelines.

Reply:
Done